code/modules/mob/living/carbon/carbon_life.dm

```diff
@@ -205,6 +207,14 @@
 		var/datum/organ/datum_organ_var_name_idk = internal_organ_datums[organ_tag]
 		datum_organ_var_name_idk.on_life()
 
+/mob/living/carbon/proc/handle_dead_organs()
+	for(var/thing in internal_organs)
+		var/obj/item/organ/internal/O = thing
+		O.on_death()
```

Comment:
this shouldnt be called `on_death`, as it implies it called once when the person dies

Reply:
Fair, was mirroring it to on_life

Reply:
renamed